### PR TITLE
Stress Test: JSON logs

### PR DESCRIPTION
**Pull request by JulianGCalderon**
Allows to save logs to a .jsonl file and plot relevant information from it

- Adds `output` flag for writing logs to a file
- Adds python script that plots the information from the output file
- Adds Makefile commands for running the stress test
- Adds documentation of the stress test in the README

I ran the test in the benchmarking server and got the following information:

server_stress_test

## Checklist
- [ ] Linked to Github Issue
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated.

### CODE REVIEW

**Comment by Oppen** [comment]
Beautiful plots! 🎉

**Review by Oppen — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ required-features = ["scarb"]
 default = ["build-cli", "with-runtime"]
 build-cli = [
   "dep:clap",
-  "dep:tracing-subscriber",
+
```
Comment (azteca1998) [inline Cargo.toml:39]: The dependency of `tracing-subscriber` should only be present for binaries, not the library. Configuring tracing is the responsibility of the binary, not the library.

Comment (JulianGCalderon) [reply Cargo.toml:39]: Is there a way to specify that a dependency is only used in a particular binary?

Comment (JulianGCalderon) [reply Cargo.toml:39]: Aah, with this: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies, right? So i should just re add dep:tracing-subscriber to the build-cli feature

Comment (JulianGCalderon) [reply Cargo.toml]: I don't remember changing that, maybe it was changed automatically by `cargo add`? Anyway, I already pushed the fix. Thanks

File: Cargo.toml [hunk 2]
```diff
@@ -90,9 +94,7 @@ cairo-lang-starknet-classes = "2.6.3"
 cairo-native-runtime = { version = "0.2.0", path = "runtime", optional = true }
 clap = { version = "4.5", features = ["derive"], optional = true }
 libloading = "0.8.3"
-tracing-subscriber = { version = "0.3", features = [
-  "env-filter",
-], optional = true }
+tracing-subscriber = { version = "0.3.18", features = ["env-filter", "json", "registry"] , optional = true}
```
Comment (azteca1998) [inline Cargo.toml:97]: Now there's a space missing after `true`.

**Review by azteca1998 — DISMISSED** [review]

File: Cargo.toml (hunk 2 above)
Comment (JulianGCalderon) [reply Cargo.toml]: Fixed

**Review by azteca1998 — APPROVED** [review]

**Review by edg-l — APPROVED** [review]

**Pull request merged**